src/richie-front/js/components/SearchSuggestField/SearchSuggestField.tsx

```diff
@@ -230,17 +243,18 @@ export class SearchSuggestFieldBase extends React.Component<
       // TypeScript incorrectly infers the type of the Autosuggest suggestions prop as SearchSuggestion, which
       // would be correct if we did not use sections, but is incorrect as it is.
       <Autosuggest
-        suggestions={suggestions as any}
+        getSectionSuggestions={suggestionsFromSection as any}
         getSuggestionValue={getSuggestionValue}
-        highlightFirstSuggestion={true}
+        highlightFirstSuggestion={value.length > 2}
```

Comment:
I found it a bit disturbing that nothing happens when I press `enter` after typing only one letter.